### PR TITLE
fix: Detailed logging for RegisterErrorCb DCHECK

### PR DESCRIPTION
**Pull request by dranikpg**

### CODE REVIEW

File: util/fibers/uring_socket.cc [hunk 1]
```diff
@@ -378,7 +378,7 @@ uint32_t UringSocket::CancelPoll(uint32_t id) {
 }
 
 void UringSocket::RegisterOnErrorCb(std::function<void(uint32_t)> cb) {
-  DCHECK(error_cb_id_ == UINT32_MAX);
+  DCHECK_EQ(error_cb_id_, UINT32_MAX) << this->fd_;
```
Comment (kostasrim) [inline util/fibers/uring_socket.cc:381]:
```suggestion
  DCHECK_EQ(error_cb_id_, UINT32_MAX) << fd_;
```

Comment (kostasrim) [reply util/fibers/uring_socket.cc:381]: no need to redirect via `this` :)

**Review by kostasrim — APPROVED** [review]

File: util/fibers/uring_socket.cc (hunk 1 above)
Comment (dranikpg) [reply util/fibers/uring_socket.cc]: Oh yes sure, I was just looking what augo-suggestion is offering me on fields

**Review by romange — APPROVED** [review]

**Comment by dranikpg** [comment]
Signed

**Pull request merged**